sdks/python/apache_beam/io/gcp/ai/video_intelligence.py

```diff
@@ -0,0 +1,75 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+A connector for sending API requests to the GCP Video Intelligence API.
+"""
+from __future__ import absolute_import
+
+from typing import Union
+
+from apache_beam import typehints
+from apache_beam.io.gcp.ai import helper
+from apache_beam.metrics import Metrics
+from apache_beam.transforms import DoFn, ParDo, PTransform
+
+
+class AnnotateVideo(PTransform):
+  """A ``PTransform`` for annotating video using the GCP Video Intelligence API
+  ref: https://cloud.google.com/video-intelligence/docs
+  """
+
+  def __init__(self, features):
```

Comment:
I think we should also accept other parameters, not only `features`.

A must have is `video_context`, because sometimes it provides additional feature-specific parameters. Others are `location_id` and possibly `metadata`. Take a look at the interface of `annotate_video()`

Reply:
Good idea, thanks! I'll have a look after we've finished the discussion with @aaltay about the usefulness of this PR.